README.md

```diff
@@ -29,6 +29,33 @@ To install from source, run:
 ```bash
 $ python3 setup.py install
 ```
+## Settings
+
+### Pointing to your Marquez service
+`marquez-airflow` needs to know where to talk to the Marquez server API.  You can set these using environment variables to be read by your Airflow service.
+
+You will also need to set the namespace if you are using something other than the `default` namespace.
+```
+MARQUEZ_HOST=my_hosted_marquez.example.com
```

Comment:
As of the most recent release `0.2.1` of `marquez-airflow`,  we switched to only `MARQUEZ_URL` for setting the Marquez backend that includes both the host and port. For example:

```
MARQUEZ_URL=http://localhost:8080
```

Reply:
👍